### PR TITLE
add doc for experimental variable tidb_store_batch_size

### PR DESCRIPTION
**Pull request by you06**
Signed-off-by: you06 <you1474600@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

ref: pingcap/tidb#39361

Document the experimental feature `tidb_store_batch_size`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -3092,6 +3092,17 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 单位：秒
 - 这个变量设置了 [statement summary tables](/statement-summary-tables.md) 的刷新时间。
 
+### `tidb_store_batch_size`
```
Comment (Oreoxmt) [inline system-variables.md:3095]:
```suggestion
### `tidb_store_batch_size` <span class="version-mark">从 v6.5.0 版本开始引入</span>
```

File: system-variables.md [hunk 2]
```diff
@@ -3092,6 +3092,17 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+
+> **警告：**
+>
+> 目前 `tidb_store_batch_size` 为实验特性，不建议在生产环境中使用。
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`0`
+- 设置 `IndexLookUp` 算子回表时 batch 多个 coprocessor task 的大小，0 代表不使用 batch。当 `IndexLookUp` 算子的回表 task 数量特别大，出现极长的慢查询时，可以适当调整该参数加速查询。
```
Comment (Oreoxmt) [inline system-variables.md:3104]:
```suggestion
- 设置 `IndexLookUp` 算子回表时 batch 多个 Coprocessor Task 的大小。`0` 代表不使用 batch。当 `IndexLookUp` 算子的回表 Task 数量特别大，出现极长的慢查询时，可以适当调整该参数加速查询。
```
这里“适当调整”可以明确为调大还是调小吗

Comment (you06) [reply system-variables.md]: 改成调大了

File: system-variables.md [hunk 3]
```diff
@@ -3092,6 +3092,18 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
 - 这个变量设置了 [statement summary tables](/statement-summary-tables.md) 的刷新时间。
 
+### `tidb_store_batch_size` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+> **警告：**
+>
+> 目前 `tidb_store_batch_size` 为实验特性，不建议在生产环境中使用。
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
```
Comment (Oreoxmt) [inline system-variables.md:3102]:
```suggestion
- 是否持久化到集群：是
- 类型：整数
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 4]
```diff
@@ -3092,6 +3092,19 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+> **警告：**
+>
+> 目前 `tidb_store_batch_size` 尚未稳定，未来此变量可能会删除，不建议业务依赖此变量或在生产环境中使用，请保留默认值。
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：整数
+- 默认值：`0`
+- 范围：`[0, 25000]`
+- 设置 `IndexLookUp` 算子回表时 batch 多个 Coprocessor Task 的大小。`0` 代表不使用 batch。当 `IndexLookUp` 算子的回表 Task 数量特别大，出现极长的慢查询时，可以适当调大该参数加速查询。
```
Comment (TomShawn) [inline system-variables.md:3106]:
```suggestion
- 设置 `IndexLookUp` 算子回表时多个 Coprocessor Task 的 batch 大小。`0` 代表不使用 batch。当 `IndexLookUp` 算子的回表 Task 数量特别多，出现极长的慢查询时，可以适当调大该参数以加速查询。
```

File: system-variables.md [hunk 5]
```diff
@@ -3092,6 +3092,19 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 单位：秒
 - 这个变量设置了 [statement summary tables](/statement-summary-tables.md) 的刷新时间。
 
+### `tidb_store_batch_size` <span class="version-mark">从 v6.5.0 版本开始引入</span>
```
Comment (TomShawn) [inline system-variables.md:3095]:
```suggestion
### `tidb_store_batch_size`
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**